packages/driver/src/cy/actionability.coffee

```diff
@@ -49,8 +50,10 @@ ensureElIsNotCovered = (cy, win, $el, fromElViewport, options, log, onScroll) ->
     ## figure out the deepest element we are about to interact
     ## with at these coordinates
     $elAtCoords = getElementAtPointFromViewport(fromElViewport)
-
+    debug('elAtCoords', $elAtCoords)
+    debug('el has pointer-events none?')
     cy.ensureElDoesNotHaveCSS($el, 'pointer-events', 'none', log)
+    debug('is descendent of elAtCoords?')
```

Comment:
I guess this is new - to send debug info for driver events. I mean, I like more events, just wondering if this is what we're advised to do now - and when do we log versus not log?